src/resources/parser/material/json-standard-material.js

```diff
@@ -167,8 +167,9 @@ class JsonStandardMaterialParser {
         if (!this._validator) {
             this._validator = new StandardMaterialValidator();
         }
-        this._validator.validate(data);
-
+        if (!data.validated) {
```

Comment:
looks like `_validate` is completely unnecessary and should be removed.

it is being called from `parse()` which immediately after invoking `validate` does `initialize()`, which handles validation anyway.

Reply:
I left it in because I thought there may be code paths that would require it? There is a similar check for the `validated` property for probably similar reasons. If you are certain there are no code paths that would require the validation I can remove it.

Reply:
I tidied up the code and removed the duplication with a call to _validate (currently required in the editor thumbnail renderer).